src/client/components/repositories/popular/Popular.js

```diff
@@ -25,16 +25,13 @@ class PopularRepositories extends React.Component {
       return <Loading />
     }
 
-    const orderedRepos = orderBy(repos, ['stargazers', 'name'], ['desc', 'asc'])
+    const orderedRepos = orderBy(repos, ['stargazers', 'forks', 'watchers', 'name'], ['desc', 'desc', 'desc', 'asc'])
 
     // Add position to each repos. Some repos will have the same amount of stars, therefore
     // we have to assign the same position for those cases.
     let currentPosition = 0
     orderedRepos.reduce((previous, current) => {
-      if (previous.stargazers !== current.stargazers) {
-        currentPosition++
-      }
-      current.position = currentPosition
+      current.position = currentPosition++
```

Comment:
@pthomas931 can you replace the `.reduce` with a `.forEach`, and instead of using the `currentPosition`, please use the `index` from the `.forEach`. :)